### PR TITLE
TK Hugs

### PR DESCRIPTION
**Pull request by jbox144**
## About The Pull Request

A quick little PR that lets you hug people with TK.

## Why It's Good For The Game

T H E Y   S H A L L   N O T   E S C A P E

## Changelog
:cl:
tweak: You can now hug people with TK
/:cl:

### CODE REVIEW

**Comment by jbox144** [comment]
yes you can headpat with this

**Comment by ZombiesVsPlants** [comment]
kek and based

**Comment by jbox144** [comment]
shit

File: austation/code/datums/mutations/telekinesis.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/mob/living/carbon/attack_tk(mob/living/carbon/owner)
+	src.help_shake_act(owner)
```
Comment (MCterra10) [inline austation/code/datums/mutations/telekinesis.dm:2]: You need . = ..() here if the parent proc should be called

Comment (MCterra10) [reply austation/code/datums/mutations/telekinesis.dm:2]: Was this tested? With various items to ensure previous functionality works

Comment (jbox144) [reply austation/code/datums/mutations/telekinesis.dm:2]: It was tested. Since carbons don't inheret any attack_tk methods, I left it as this.

**Comment by MCterra10** [comment]
I think this is ok

**Pull request merged**